books/free-programming-books-langs.md

```diff
@@ -528,6 +528,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Software Design Using C++](http://cis.stvincent.edu/html/tutorials/swd/) - Br. David Carlson, Br. Isidore Minerd
 * [Software optimization resources](http://www.agner.org/optimize/) - Agner Fog
 * [The Boost C++ libraries](http://theboostcpplibraries.com) - Boris Schäling (HTML)
+* [The C++ Programming Language - 4th Edition by Bjarne Stroustrup(Creator of C++)](http://chenweixiang.github.io/docs/The_C++_Programming_Language_4th_Edition_Bjarne_Stroustrup.pdf) - Chenweixiang (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines. remove it

Reply:
Oh, sure